install.sh

```diff
@@ -0,0 +1,13 @@
+cd matrix
+echo "Running rgbmatrix installation..."
+sudo apt-get update && sudo apt-get install python2.7-dev python-pillow -y
+make build-python
+sudo make install-python
+cd bindings
+sudo pip install -e python/
+cd ../../
+echo "Installing required dependencies. This may take some time (10 minutes-ish)..."
+sudo apt-get install libxml2-dev libxslt-dev
+sudo pip install mlbgame pytz tzlocal
```

Comment:
Update this to install version 2.4.2 since the most recently tagged version of mlbgame breaks all of the things.